lmdeploy/turbomind/engine_config.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from pydantic.dataclasses import dataclass
+
+
+@dataclass
+class EngineConfig:
```

Comment:
Shall we update this entrypoint function so as to run `lmdeploy chat turbomind <model_name_or_path> <turbomind_engine_config>` in lmdeploy0.2.0

https://github.com/InternLM/lmdeploy/blob/755c1d0064a0f862961819446978ec1241873461/lmdeploy/turbomind/chat.py#L32

Reply:
For example, add a `engine_config` argument